packages/nibijs/src/chain/chain.ts

```diff
@@ -136,12 +145,23 @@ export const Devnet = (chainNumber: number) =>
     number: chainNumber,
   })
 
+/** Mainnet: "Chain" configuration for the Nibiru "mainnet".
+ * ❗ Mainnet uses real funds. For more info, see
+ * <a href="https://nibiru.fi/docs/dev/networks/">Nibiru Netwokrs</a>.
+ * */
+export const Mainnet = (chainNumber: number = 1) =>
```

Comment:
Unrelated to this change but should we also add the heart monitor & the web socket endpoint as apart of the chain interface?

We currently create those urls in the web app

Reply:
https://github.com/NibiruChain/ts-sdk/pull/269